gcc/jit/docs/topics/types.rst

```diff
@@ -436,6 +436,11 @@ Reflection API
 
      Return non-zero if the type is an integral.
 
+.. function::  int\
+               gcc_jit_type_is_float (gcc_jit_type *type)
+
+     Return non-zero if the type is a float.
```

Comment:
Now that I see this, I believe we should rename the function to make sure people do not think that this checks that this is the `float` type in C.
I would rename every `float` to `floating_point` (for instance `gcc_jit_type_is_floating_point`) unless you have a better idea.

Reply:
Yes, let's change it to `gcc_jit_type_is_floating_point()`